media/apps.html

```diff
@@ -2,23 +2,74 @@
+          <ul>
+            <li><a data-featureid="navigation-timing">Navigation Timing</a> exposes timing information related to navigation and elements;</li>
+            <li><a data-featureid="resource-timing">Resource Timing</a> exposes timing information for resources in a document;</li>
+            <li><a data-featureid="user-timing">User Timing</a> help applications measure the performance of their applications using high precision timestamps.</li>
+          </ul>
+        </div>
+      </section>
+      <section class="featureset in-progress">
+        <h2>Technologies in progress</h2>
+        <div data-feature="Detecting Capabilities">
+          <p>To improve the user experience and take advantage of advanced device capabilities when they are available, media providers need to know the decoding (and encoding) capabilities of the user's device. Can the device decode a particular codec at a given resolution, bitrate and framerate? Will the playback be smooth and power efficient? Can the display render high dynamic range (HDR) and wide color gamut content? The <a data-featureid="media-capabilities">Media Capabilities</a> specification defines an API that helps application developers determine device capabilities. The API aims at replacing the more basic and vague <code>canPlayType()</code> and <code>isTypeSupported()</code> functions defined in HTML and Media Source Extensions (MSE).</p>
+          <p>On top of getting answers to <em>"Can"</em> questions, media providers also want answers to "<em>Should</em>" questions, such as "Should I sent HDR content if I have both HDR and SDR variants?". The Media Capabilities specification does not answer these questions, which rather fall within the realm of CSS extensions describing the capability of the display. Notably, <a data-featureid="color-gamut">CSS Media Queries Level 4</a> includes means to detect wide-gamut displays and adapt the rendering of the application to them. <a data-featureid="mediaqueries5">CSS Media Queries Level 5</a>, to be published after Level 4, will introduce a <code>video-</code> prefix to some features to detect differences between the video plane and the graphics plane on devices that render video separately from the rest of an HTML document, and that e.g. may support High Dynamic Range (HDR) for video and only Standard Dynamic Range (SDR) for other types of content.</p>
```

Comment:
Suggest: sent -> send

Reply:
Thanks, I also fixed the same typo in the "distribution" page.